addusers.py

```diff
@@ -336,7 +337,7 @@ def parse_rows(rows):
     session = session.Session(auth=auth)
     
     openstack = Openstack(session=session, nova_version=nova_version, setpass_url=setpass_url)
-    auth_file = config.get("excelsheet", "auth_file")
+    auth_file = get_absolute_path(config.get("excelsheet", "auth_file"))
     worksheet_key = config.get("excelsheet", "worksheet_key")
```

Comment:
So, just want to make sure, this also needs a `get_absolute_path` right? But not within this PR :)